pallets/phala/src/compute/computation.rs

```diff
@@ -1090,6 +1108,19 @@ pub mod pallet {
 		}
 	}
 
+	const FIXED_1E12: FixedPoint = fixed_macro::fixed!(1_000_000_000_000: U64F64);
+	pub fn balance_to_fixed(b: u128) -> FixedPoint {
+		let v = FixedPoint::from_num(b);
+		v.saturating_div(FIXED_1E12)
+	}
+	pub fn balance_from_fixed(v: FixedPoint) -> u128 {
+		v.saturating_mul(FIXED_1E12).to_num()
+	}
+	pub fn balance_from_bits(bits: u128) -> u128 {
+		let v = FixedPoint::from_bits(bits);
+		balance_from_fixed(v)
+	}
```

Comment:
This piece of code is copied from the `balance_convert.rs`. Could it be merged?
And should we let it panic on overflow?